stage0/gc.go

```diff
@@ -55,8 +55,12 @@ func GC(pdir string, uuid *types.UUID) error {
 	if debugEnabled {
 		args = append(args, "--debug")
 	}
+	if localConfig != "" {
+		args = append(args, "--local-config", localConfig)
```

Comment:
This looks overly generic and unclear. It seems to expect the directory (not the actual config file) for CNI, right? If so, something like `--cni-config-dir` sounds better. Unless we are planning to use this more generically later, in which case `--config-dir` may be ok.